src/frontend/src/flows/manage/index.ts

```diff
@@ -115,36 +115,65 @@ const anchorSection = (userNumber: bigint): TemplateResult => html`
 const devicesSection = (
   devices: DeviceData[],
   onAddDevice: (next: "canceled" | "local" | "remote") => void
-): TemplateResult => html`
-    <aside class="l-stack">
-      <div class="t-title t-title--complications">
-        <h2 class="t-title">Added devices</h2>
-        <span class="t-title__complication c-tooltip">
-          <span class="c-tooltip__message c-card c-card--narrow">
-            You can register up to ${MAX_AUTHENTICATORS} authenticator
-            devices (recovery devices excluded)</span>
-            (${numAuthenticators(devices)}/${MAX_AUTHENTICATORS})
-          </span>
-        </span>
-        <button
-          ?disabled=${numAuthenticators(devices) >= MAX_AUTHENTICATORS}
-          class="t-title__complication t-title__complication--end c-tooltip c-tooltip--onDisabled"
-          @click="${async () => onAddDevice(await chooseDeviceAddFlow())}"
-          id="addAdditionalDevice"
-        >
-          <span class="c-tooltip__message c-tooltip__message--right c-card c-card--narrow"
-            >You can register up to ${MAX_AUTHENTICATORS} authenticator devices.
-            Remove a device before you can add a new one.</span
-          >
-          <span class="t-link t-link--discreet">
-            <i class="t-link__icon" aria-hidden=true>+</i>
-            Add new device
+): TemplateResult => {
+  const wrapClasses = ["l-stack"];
+  const isWarning = devices.length < 2;
+
+  if (isWarning === true) {
+    wrapClasses.push("c-card", "c-card--narrow", "c-card--warning");
+  }
+
+  return html`
+    <aside class="${wrapClasses.join(" ")}">
+      ${
+        isWarning === true
```

Comment:
here and elsewhere you can just have `isWarning` without `=== true`, our typescript config will make sure you never check for anything truthy instead of a boolean

Reply:
oh OK i guess I am used to do it, do you want me to remove it?

Reply:
All good!